### PR TITLE
sign-up button typo fixed

### PR DESCRIPTION
**Pull request by yanabar**
typo now fixed

### CODE REVIEW

File: pimpmycause/pimpmycause/templates/index.html [hunk 1]
```diff
@@ -353,7 +353,7 @@ <h4 class="pnl__title">Sign-up to our newsletter</h4>
                                         <input type="email" name="email" placeholder="Email Address" required>
                                     </div>
                                     <div class="medium-6 large-6 small-12 columns">
-                                        <input class="button" type="submit" value="Singup">
+                                        <input class="button" type="submit" value="Sign-up">
```
Comment (lili2311) [inline pimpmycause/pimpmycause/templates/index.html]: After some furious googling seems it is actually "Sign up" :D Do you mind making the small change?

Comment (yanabar) [inline pimpmycause/pimpmycause/templates/index.html]: I had the same dilemma..  :) Do i leave the one below as it is though or change it also?
image

Comment (lili2311) [inline pimpmycause/pimpmycause/templates/index.html]: yeah also change 🙈

**Comment by yanabar** [comment]
Done!

**Pull request merged**